README.md

```diff
@@ -1,119 +1,160 @@
+
+> Here is an example of a YAML block for a region or a department:
+
 ```yaml
-  nom: region-ou-departement-exemple
-  code: Exemple
-  casConfirmes: 500
-  gueris: 40 # valeur copiée du fichier YAML précédent
-  deces: 10
-  hospitalises: 10
-  reanimation: 5
+  nom: region-ou-departement-exemple # region-or-departement-example
+  code: Exemple                      # Example
+  casConfirmes: 500                  # confirmed cases
+  gueris: 40 # valeur copiée du fichier YAML précédent / recovered: value copied from the previous YAML file
+  deces: 10                          # deaths
+  hospitalises: 10                   # hospitalized
+  reanimation: 5                     # reanimated
   victimes:
     - age: 85
       date: 2020-03-10
-      sexe: homme
-    - sexe: femme
+      sexe: homme                    # sex: male
+    - sexe: femme                    # sex: female
       date: 2020-03-10
     - date: 2020-03-10
 ```
 
 Les champs `casConfirmes`, `gueris` et `deces` comptabilisent le total par catégorie depuis le début de la crise Covid-19. Par contre, les champs `hospitalises` et `reanimation` donnent le nombre de patient par catégorie à l'instant de l'édition du bulletin d'information, ces 2 chiffres peuvent bien sûr évoluer à la hausse ou à la baisse.
+> The fields `casConfirmes`,` gueris` and `deces` count the total by category since the start of the Covid-19 crisis. On the other hand, the fields `hospitalized` and` resuscitation` give the number of patients by category at the time of publication of the newsletter, these 2 figures can of course go up or down.
 
 Notez qu'il est possible si besoin d'ajouter des commentaires en fin de ligne en utilisant le caractère `#`
+> Note that it is possible if necessary to add comments at the end of the line using the character `#`
 
 Le bloc `victimes` détaille les informations du bulletin concernant les personnes décédés (et non les personnes contaminées). Attention ce champ ne comptabilise pas toutes les victimes depuis le début de la crise, mais uniquement les victimes annoncées dans le bulletin. Pour chaque victime, on ajoute un tiret `-`, puis les informations sur la personne. Si aucune information, ajoutez la date du décès `- date: 2020-03-10`. Si vous disposez de plus d'information, ajoutez un tiret `-` par victime puis toutes les informations disponibles `age`, `sexe` et/ou `date` (cf. exemple ci-dessus)
+> The `victims` block details the information in the bulletin concerning the deceased (and not the infected). Please note that this field does not count all the victims since the start of the crisis, but only the victims announced in the bulletin. For each victim, we add a dash `-`, then the information about the person. If you have no information, add the date of death `- date: 2020-03-10`. If you have more information, add a dash `-` per victim, then all the information available` age`, `sex` and / or` date` (see example above)
```

Comment:
You translated the field names as well when code quoting

```suggestion
> The `victimes` block details the information in the bulletin concerning the deceased (and not the infected). Please note that this field does not count all the victims since the start of the crisis, but only the victims announced in the bulletin. For each victim, we add a dash `-`, then the information about the person. If you have no information, add the date of death `- date: 2020-03-10`. If you have more information, add a dash `-` per victim, then all the information available` age`, `sexe` and / or` date` (see example above)
```